monome/teletype
# W/ i2c operators

#### What does this PR do?
Adds support for Mannequins W/ module via i2c. Provides getters & setters for 4 parameters of the W/ module.

#### Provide links to any related discussion on [lines](https://llllllll.co/).

#### How should this be manually tested?
Call `WS.PLAY 1` and you should see/ hear the W/ module start playback forward. `WS.PLAY 0` to stop. Currently getters are only included in the W/ beta firmware 1.1.0b+.

#### Any background context you want to provide?

#### If the related Github issues aren't referenced in your commits, please link to them here.

#### I have,
